Dockerfile

```diff
@@ -1,14 +1,17 @@
-FROM alpine
+FROM alpine:3.19.1
```

Comment:
I would suggest to use  `python:3-alpine`, then the 2 `apk` lines become unnecessary, I think.

Reply:
Good idea. I will tackle this in a separate PR.